tidb-resource-control.md

```diff
@@ -0,0 +1,138 @@
+- You can choose to combine all test environments into a single cluster, or group the batch tasks that consume more resources into a single resource group. It can improve hardware utilization and reduce operating costs while ensuring that critical applications receive the necessary resources.
+
+In addition, the rational use of the resource control feature can reduce the number of clusters, ease the difficulty of operation and maintenance, and save management costs.
+
+<CustomContent platform="tidb-cloud">
+
+> **Note:**
+>
+> This feature does not support [Serverless Tier clusters](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta).
+
+</CustomContent>
+
+## Parameters for resource control
+
+The resource control feature introduces two new global variables.
+
+* TiDB: you can use the system variable [`tidb_enable_resource_control`](/system-variables.md#tidb-tidb_enable_resource_control) to control whether to enable flow control for a resource group. 
+* TiKV: you can use the parameter [`resource_control.enabled`](/tikv-configuration-file.md#resource_control) to control whether to use request scheduling based on resource group quotas. 
+
+<CustomContent platform="tidb-cloud">
+
+> **Note:**
+>
+> The parameter `resource_control.enabled` is disabled by default. It does not support dynamic modification. You need to contact the [PingCAP support team](/tidb-cloud/tidb-cloud-support.md) to enable it. You need to restart the TiKV instance for the modification to take effect.
+
+</CustomContent>
+
+The results of the combination of these two parameters are shown in the following table.
+
+| `resource_control.enabled`  | `tidb_enable_resource_control`= ON   | `tidb_enable_resource_control`= OFF  |
+|:----------------------------|:-------------------------------------|:-------------------------------------|
+| `resource_control.enabled`= true  |  Flow control and scheduling (recommended) | Invalid combination      |  
+| `resource_control.enabled`= false |  Only flow control                         | The feature is disabled. |
+
+## How to use resource control
+
+For an existing resource group, you can modify the read and write quota of the resource group by using [`ALTER RESOURCE GROUP`](/sql-statements/sql-statement-alter-resource-group.md). The quota changes to the resource group take effect immediately.
+
+You can delete a resource group by using [`DROP RESOURCE GROUP`](/sql-statements/sql-statement-drop-resource-group.md). The user bound by the deleted resource group will use the `default` resource group for resource isolation.
+
+> **Note:**
+> 
+> - When you bind a user resource group with `CREATE USER` or `ALTER USER`, it will not take effect for the user's existing sessions, but only for the user's new sessions.
```

Comment:
```suggestion
> - When you bind a user to a resource group by using `CREATE USER` or `ALTER USER`, it will not take effect for the user's existing sessions, but only for the user's new sessions.
```